packages/typespec-test/test/openai_generic/generated/typespec-ts/src/api/audio/translations/index.ts

```diff
@@ -31,7 +32,7 @@ export function _createSend(
       ...operationOptionsToRequestParameters(options),
       contentType: (options.contentType as any) ?? "multipart/form-data",
       body: {
-        file: audio["file"],
+        file: uint8ArrayToString(audio["file"], "base64"),
```

Comment:
This is because we have to remove the encode in the typespec as the tcgc linter rule. And previously we have agreed that the default encoding is base64 for bytes. As we will support multipart/form-data later, I would prefer we don't do extra effort here. @joheredi what do you think?

Reply:
Yes! we should have customers do the encoding explicitly in RLC